addons/sale/models/payment.py

```diff
@@ -61,33 +61,40 @@ def _set_transaction_pending(self):
         super(PaymentTransaction, self)._set_transaction_pending()
 
         for record in self:
-            sales_orders = record.sale_order_ids.filtered(lambda so: so.state == 'draft')
-            sales_orders.force_quotation_send()
+            sales_orders = record.sale_order_ids.filtered(lambda so: so.state in ['draft', 'sent'])
+            # set quote as sent when transaction is pending
+            # but do not send mail because mail will be send based on configuration in setting.
```

Comment:
I don't really understand this comment.
I think it can be removed.

Reply:
Done